tests/alive-tv/memory/punning-ptrint.tgt.ll

```diff
@@ -0,0 +1,8 @@
+target datalayout = "e-m:o-i64:64-f80:128-n8:16:32:64-S128"
+
+define i8 @ptr_int_punning(i8** %pptr, i64 %n) {
+  %ptr = call i8* @malloc(i64 1)
+  store i8* %ptr, i8** %pptr
+  ret i8 0
```

Comment:
I think ret undef would be better here (since malloc can return null, 0 isn't a particularly good value).

Reply:
Updated, and added malloc null check as well

Reply:
(This comment was originally attached to BB2's ret, but dislocated after the update)